src/main/resources/avro/metadata.avdl

```diff
@@ -9,6 +9,32 @@ protocol Metadata {
 import idl "common.avdl";
 
 /**
+  An ontology term describing an attribute. (e.g. the phenotype attribute
+  'polydactyly' from HPO)
+  */
+  record OntologyTerm {
```

Comment:
@sarahhunt Good question; we're currently (i.e. last/this week) discussing the ```OntologyTerm``` structure, to stabilize & document it. That's an item where other people (probably) have strong opinions, and since it is related to many records we have to get this with a good consensus score. Pls. follow up on this!